src/vs/base/browser/dom.ts

```diff
@@ -1398,7 +1398,12 @@ function toBinary(str: string): string {
 	for (let i = 0; i < codeUnits.length; i++) {
 		codeUnits[i] = str.charCodeAt(i);
 	}
-	return String.fromCharCode(...new Uint8Array(codeUnits.buffer));
+	let binary = '';
+	const uint8array = new Uint8Array(codeUnits.buffer);
+	for (let i = 0; i < uint8array.length; i++) {
+	    	binary += String.fromCharCode(uint8array[i]);
```

Comment:
fyi there are some extra spaces here which cause the hygiene check to fail when doing the official build on linux, I just made a commit in master for it c643c433e82168791d5ff0082703546581307a80

Reply:
Thanks @RMacfarlane! Just pushed a fix here too